app/src/main/java/io/apicurio/registry/mt/TenantContext.java

```diff
@@ -23,16 +23,22 @@
  */
 public interface TenantContext {
 
-    public static final String DEFAULT_TENANT_ID = "_";
+    String DEFAULT_TENANT_ID = "_";
```

Comment:
why change this from static field to object field? I think this change is not a good idea